### PR TITLE
Add 'bytesRead' and 'path' to fs.ReadStream

### PR DESCRIPTION
**Pull request by lukas-zech-software**
Please fill in this template.

- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] The package does not provide its own types, and you can not add them.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).

If changing an existing definition:
- [x] Provide a URL to  documentation or source code which provides context for the suggested changes: 

https://nodejs.org/api/fs.html#fs_readstream_bytesread
https://nodejs.org/api/fs.html#fs_readstream_path

- [ ] Increase the version number in the header if appropriate.

### CODE REVIEW

**Comment by yuit** [comment]
Thank you for the PR @lukas-zech-software . The change looks good. The build break is fixed in anther branch. Once that branch is merged, could you update your PR as well?

File: node/index.d.ts [hunk 1]
```diff
@@ -2019,6 +2019,8 @@ declare module "fs" {
     export interface ReadStream extends stream.Readable {
         close(): void;
         destroy(): void;
+        bytesRead: number;
```
Comment (Kovensky) [inline node/index.d.ts]: https://nodejs.org/dist/latest-v7.x/docs/api/fs.html#fs_readstream_bytesread

File: node/index.d.ts [hunk 2]
```diff
@@ -2019,6 +2019,8 @@ declare module "fs" {
...
+        path: string | Buffer;
```
Comment (Kovensky) [inline node/index.d.ts]: https://nodejs.org/dist/latest-v7.x/docs/api/fs.html#fs_readstream_path

File: node/node-tests.ts [hunk 3]
```diff
@@ -423,6 +423,8 @@ function stream_readable_pipe_test() {
     var r = fs.createReadStream('file.txt');
     var z = zlib.createGzip({ finishFlush: zlib.Z_FINISH });
     var w = fs.createWriteStream('file.txt.gz');
+    console.log(r.bytesRead);
+    console.log(r.path);
```
Comment (Kovensky) [inline node/node-tests.ts]: These check more for the existence than for the type correctness; not sure what the guidelines are for this.

Comment (lukas-zech-software) [inline node/node-tests.ts]: You're right, I wasn't sure what these test should actually achieve.

I'll add tests to check the actually types during runtime

**Comment by andy-ms** [comment]
@lukas-zech-software Please merge from the latest `types-2.0` to fix tests.

**Comment by lukas-zech-software** [comment]
@yuit @andy-ms I merged the latest changes, but the tests still fail.

I cannot see why they fail though

**Comment by andy-ms** [comment]
I've run tests manually. In the future this should be fixed by #12698.

**Pull request merged**